src/NLog.Extensions.Logging/Logging/NLogMessageParameterList.cs

```diff
@@ -47,10 +44,9 @@ public static NLogMessageParameterList TryParse(IReadOnlyList<KeyValuePair<strin
         /// <summary>
         /// Verify that the input parameterList contains non-empty key-values and the orignal-format-property at the end
         /// </summary>
-        private static bool IsValidParameterList(IReadOnlyList<KeyValuePair<string, object>> parameterList, out int? originalMessageIndex, out bool customCaptureTypes, out bool isPositional)
+        private static bool IsValidParameterList(IReadOnlyList<KeyValuePair<string, object>> parameterList, out int? originalMessageIndex, out bool requiresNLogParser)
```

Comment:
requiresNLogParser, is this maybe 'requiresStructuredLogging` / `requiresStructuredMessageParsing`?

Reply:
Microsoft Extension Logging supports structured logging. It is the home-invented message-template notation that is not support and requires help from NLog (or Serilog).

Reply:
Force push made with new name: `HasMessageTemplateCapture`

Reply:
sounds good to me